packages/terra-modal/src/ModalContent.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import FocusTrap from 'focus-trap-react';
+import ModalOverlay from './ModalOverlay';
+
+const propTypes = {
+  ariaLabel: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  classNameModal: PropTypes.string,
+  classNameOverlay: PropTypes.string,
+  closeOnOutsideClick: PropTypes.bool,
+  closePortal: PropTypes.func,
+  role: PropTypes.string,
+};
+
+const defaultProps = {
+  ariaLabel: null,
+  children: null,
+  classNameModal: null,
+  classNameOverlay: null,
+  closeOnOutsideClick: true,
+  role: 'dialog',
+};
+
+
+class ModalContent extends React.Component {
+
+  componentDidMount() {
+    // Disable background scrolling
+    document.body.style.overflow = 'hidden';
```

Comment:
I'm not sure I quite follow. Are you proposing that when the modal is active, content in the background can still scroll? 

Also, the current implementation enforces that there is always an overlay when a modal is active.